onnxruntime/core/providers/dml/DmlExecutionProvider/src/GraphTransformer.cpp

```diff
@@ -84,14 +84,20 @@ namespace Dml
         // graph while iterating over it
         std::vector<NodeToAdd> nodesToAdd;
 
+        const auto kernel_type_str_resolver = onnxruntime::AutoRegisteringKernelTypeStrResolver{};
+        gsl::not_null<const onnxruntime::KernelRegistry*> registry = m_providerImpl->GetKernelRegistry().get();
+        const auto kernel_lookup = onnxruntime::KernelLookup{onnxruntime::kDmlExecutionProvider,
```

Comment:
fix this to not take a reference to temporary std::string created from onnxruntime::kDmlExecutionProvider